### DISCUSSION TITLE
尝试下载直播流错误

### DISCUSSION THREAD

**Discussion by BLIZZAED** [HMBSbige/BilibiliLiveRecordDownLoader#239] [Q&A]
开始录制后日志显示：“尝试下载直播流错误 The request was canceled due to the configured HttpClient.Timeout of 3 seconds elapsing.”
录制状态显示启动中
![)202UHMYGP$V~L_YV9Z0}C1](https://user-images.githubusercontent.com/108011552/175060630-3fcbd203-a1f7-4d84-b286-87d0ade2c636.png)
![1(`3{090 U@I4 (~8`VL(VD](https://user-images.githubusercontent.com/108011552/175060817-88e5fe86-a1a5-484c-a226-491807a18712.png)
有尝试过换成旧版本、换32位、换网络、更改画质，依然会有此问题，有朋友遇见过吗？

**Comment by BLIZZAED** [comment]
以及无法显示主播录
![BRCXM@L `6XWBC4IF4JUQ~6](https://user-images.githubusercontent.com/108011552/175062910-a9676df7-be11-42b1-913e-f36aef373c93.png)
播

**Reply by HMBSbige** [reply]
https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/discussions/133#discussioncomment-1209763

**Reply by BLIZZAED** [reply]
请问无法录制直播可能是哪方面出了问题

**Reply by HMBSbige** [reply]
一般是自己网或者B站的问题，或者就是单纯主播开播但没有推流，具体什么问题还得你自己排查了
